### DISCUSSION TITLE
Automatically open PRs to bump cask versions with bot account

### DISCUSSION THREAD

**Discussion by AnqiPang** [Homebrew/discussions#4132] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 3.6.20
ORIGIN: https://github.com/Homebrew/brew
HEAD: 344d32bf7f40183359592ef81ca6486e0587981b
Last commit: 6 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: b9f9d4cef5fa430eb03c4ce25299ced3a151e70c
Core tap last commit: 5 days ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_GITHUB_API_TOKEN: set
HOMEBREW_MAKE_JOBS: 12
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: dodeca-core 64-bit kabylake
Clang: 14.0.0 build 1400
Git: 2.38.0 => /usr/local/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.6.1-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  yq
  docker-credential-helper-ecr
  awscli
  vde
  kubernetes-cli
```

### Description of issue

Hi,
I'm a developer for a project distributed via homebrew-cask and looking for a way to automatically bump the cask version after each release. I'm considering using `bump-cask-pr` in a github workflow to open a PR from a fork of homebrew-cask with GitHub account. Is it acceptable for homebrew to receive PRs opened by a bot user from a organization fork of homebrew-cask? And I'm curious whether homebrew-cask has a mechanism to fetch the latest version from source regularly if the developers don't update the cask. 
Any suggestions would be appreciated!

**Comment by SMillerDev** [comment, marked as answer]
>  Is it acceptable for homebrew to receive PRs opened by a bot user from a organization fork of homebrew-cask? 

Yes, that's fine. Some organizations already do this.
> And I'm curious whether homebrew-cask has a mechanism to fetch the latest version from source regularly if the developers don't update the cask.

Other than some volunteers doing updates to many casks. No, not really
